cranelift/codegen/src/isa/zkasm/lower/isle.rs

```diff
@@ -249,7 +249,9 @@ impl generated_code::Context for ZkAsmIsleContext<'_, '_, MInst, ZkAsmBackend> {
         let insts = match ty {
             F32 => MInst::load_fp_constant32(tmp, val as u32, alloc_tmp),
             F64 => MInst::load_fp_constant64(tmp, val, alloc_tmp),
-            _ => MInst::load_constant_u64(tmp, val, alloc_tmp),
+            I32 => MInst::load_constant_u32(tmp, val, alloc_tmp),
+            I64 => MInst::load_constant_u64(tmp, val, alloc_tmp),
+            _ => panic!("Not implemented"),
```

Comment:
Changed it to ``panic`` because I think it is not very good when it don't find an appropriate type and silently changed the type to ``I64``.